client/src/components/Payment.jsx

```diff
@@ -60,43 +83,136 @@ class Payment extends React.Component {
+                value={this.state.note}
+                onChange = {this.handleInputChanges.bind(this)}
+                hintText="Leave a comment"
+                floatingLabelText="Got something to say?"
+                fullWidth={true}
+                multiLine={true}
+              /><br />
+            </div>
+          </label>
+        </div>
+
+        <div className="pay-button-container"> 
+          {this.state.paymentFail
+            ? <label className='payment-fail'>
+                Error in payment processing
+              </label>
+            : null
+          }
+
+          <div>
+            <FlatButton 
+              className='pay-button' 
+              onClick={this.payUser.bind(this)} 
+              label="Pay!" 
+              primary={true} 
+              style={style.button} 
+              backgroundColor="#3D95CE"
+              hoverColor='#03A9F4'
+              labelStyle={style.button.lable}
+            />
+          </div>
+
+        </div>
+
+      </Paper>
     );
   }
 }
 
+  // render() {
+  //   return (
+  //     <Paper className='payment-container' style={style.form}>
+  //       <div className='payment-item-container'>
+  //         <label>
+  //           To Pay:
+  //           <div className="form-box top left">
+  //           <input
+  //             name='payeeUsername'
+  //             value={this.state.payeeUsername}
+  //             onChange = {this.handleInputChanges.bind(this)}
+  //           />
+  //           </div>
+  //           <br />
+  //         </label>
+  //         <label>
+  //           Amount:
+  //           <div className="form-box top right">
+  //           <input
+  //             name='amount'
+  //             value={this.state.amount}
+  //             onChange = {this.handleInputChanges.bind(this)}
+  //           />
+  //           </div>
+  //           <br />
+  //         </label>
+  //         <label>
+  //           <div className="form-box">
+  //           <input
+  //               name='note'
+  //               value={this.state.note}
+  //               onChange = {this.handleInputChanges.bind(this)}
+  //           />
+  //           </div>
+  //           <br />
+  //         </label>
+  //       </div>
+
+  //       <div className="pay-button-container"> 
+  //         {this.state.paymentFail
+  //           ? <label className='payment-fail'>
+  //               Error in payment processing
+  //             </label>
+  //           : null
+  //         }
+  //         <div>
+  //           <FlatButton 
+  //             className='pay-button' 
+  //             onClick={this.payUser.bind(this)} 
+  //             label="Pay!" 
+  //             primary={true} 
+  //             style={style.button} 
+  //             backgroundColor="#3D95CE"
+  //             hoverColor='#03A9F4'
+  //             labelStyle={style.button.lable}
+  //           />
+  //         </div>
+  //       </div>
+  //     </Paper>
+  //   );
+  // }
+
```

Comment:
Remove commented out code, unless you want to keep this for some reason?

Reply:
I am keeping it incase I need to change the styling. I'll remove it when the final decision is made on the payment component styling.